src/tests/components/Location.test.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LocationDetails from "../../components/Location";
+import {
+  kelvinToCelsius,
+  timeConvert,
+  longDateFromTime,
+} from "../../components/DataFunction";
+
+describe("LocationDetails", () => {
+  const location = {
```

Comment:
By convention this should be an object called validProps with a property called location containing the relevant data